index.html

```diff
@@ -15,7 +15,31 @@
 </head>
 
 <body>
+ 
+    <nav>
+      <h2>Fun Bus</h2>
+
+      <div class="navigation">
+        <a href="#">Home</a>
+        <a href="#">About Us</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+      </div>
+    </nav>
+
 
+<header> 
+    <div class="header-img">
+      <img src="img/fun-bus.jpg">  
```

Comment:
Div isn't necessary here since you're not manipulating multiple elements within the div. You could just manipulate the image itself.